src/Commands/Sync.php

```diff
@@ -86,7 +88,10 @@ public function handle(Backoff $backoff, SubstrateWebsocket $rpc): int
                 $this->error($message = __('enjin-platform::error.line_and_file', ['line' => $e->getLine(), 'file' => $e->getFile()]));
                 PlatformSyncError::dispatch($message);
             })
-            ->run(fn () => $this->startSync($rpc));
+            ->run(function () use ($rpc): void {
+                Cache::forget(PlatformCache::CUSTOM_TYPES->key());
```

Comment:
Can't this be added inside `startSync` function?

Reply:
Sure, done